src/webui/webapplication.cpp

```diff
@@ -129,6 +128,18 @@ namespace
 
         return languages.join(u'\n');
     }
+
+    bool isValidCookieName(const QString &cookieName)
+    {
+        if (cookieName.isEmpty() || (cookieName.size() > 128))
+            return false;
+
+        const QRegularExpression invalidNameRegex {u"[^a-zA-Z0-9_\\-]"_qs};
```

Comment:
This is a very general remark and I don't know if this has been discussed already.
For regex we should consider switching to [raw string literals](https://en.cppreference.com/w/cpp/language/string_literal). It will improve regex readability by dropping the need for backslash escapes.
However, I don't know if raw string literals can mix with user-defined literals (`_qs`).

Reply:
> I don't know if this has been discussed already.
> For regex we should consider switching to [raw string literals](https://en.cppreference.com/w/cpp/language/string_literal).

We didn't discuss such a requirement earlier. 


> It will improve regex readability by dropping the need for backslash escapes.

Personally, I don't care about it in trivial cases like this. In complex regexps, it really makes sense.
However, if you see the point of making this a requirement for regular expressions, I don't mind. But in this case, it would be good to start by converting all existing instances, so that later some newly-minted contributors do not poke their finger there when they are pointed out to such a requirement.

Reply:
I am not keen into make it a hard requirement. You should treat it more like a suggestion IMO.
Do you think it should be a requirement?

Reply:
> Do you think it should be a requirement?

No.